src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -208,8 +208,33 @@ runtime::TrtEngineAndContext TensorRTBuilder::BuildEngine(
   CHECK_EQ(engine->getNbBindings(),
            num_input_bindings + network_output_names_.size());
   nvinfer1::IExecutionContext* context = engine->createExecutionContext();
-  return {engine, context, network_input_names_, network_input_is_baked_,
-          network_output_names_};
+  std::vector<runtime::NDArray> device_buffers;
+  for (int i = 0; i < network_input_names_.size(); i++) {
+    if (network_input_is_baked_[i]) continue;
+    if (execution_args_[i]->ctx.device_type == kDLGPU) {
+      device_buffers.emplace_back();
+    } else {
+      std::vector<int64_t> shape_(execution_args_[i]->shape,
```

Comment:
Names ending in `_` is reserved for member variables, let's use `shape` instead

Reply:
done,